bee-api/bee-rest-api/src/types/dtos.rs

```diff
@@ -1484,16 +1480,8 @@ impl TryFrom<&FoundryOutputDto> for FoundryOutput {
                     .map_err(|_| Error::InvalidField("token_tag"))?;
                 decoded_token_tag
             },
-            value
-                .circulating_supply
-                .0
-                .parse::<U256>()
-                .map_err(|_| Error::InvalidField("circulating_supply"))?,
-            value
-                .maximum_supply
-                .0
-                .parse::<U256>()
-                .map_err(|_| Error::InvalidField("maximum_supply"))?,
+            U256::from_dec_str(value.circulating_supply.0).map_err(|_| Error::InvalidField("circulating_supply"))?,
```

Comment:
I would also assume that U256::to_string uses hex

Reply:
Let me check the code

Reply:
Seems to be fine as it's now
```Rust
let native_tokens = NativeTokenDto::try_from(&NativeToken::new(TokenId::new(token_id), U256::from(50))?).unwrap();
println!("{}", U256::from(50));
println!("{:?}", native_tokens);
println!("{}", serde_json::to_string(&native_tokens)?);
```
prints
```bash
50
NativeTokenDto { token_id: TokenIdDto("08e68f7616cd4948efebc6a77c4f93aed770ac53860100000000000000000000000000000000"), amount: U256Dto("50") }
{"id":"08e68f7616cd4948efebc6a77c4f93aed770ac53860100000000000000000000000000000000","amount":"50"}
```

Reply:
Doesn't make any sense :trollface:
Finishing a PR, I'll look in a bit.

Reply:
For serialization it also uses hex, but in the DTO it's only a String and there we use `.to_string()`
https://github.com/iotaledger/bee/blob/aacf78d48d01d36cadacb2f4b8acc7e9c1922ae3/bee-api/bee-rest-api/src/types/dtos.rs#L880

Reply:
Ah alright, spoke too quickly, my bad thanks 👍